src/dotnet/project.json

```diff
@@ -9,13 +9,23 @@
         "commands/dotnet-new/CSharp_Console/project.json.template",
         "commands/dotnet-new/FSharp_Console/NuGet.config",
         "commands/dotnet-new/FSharp_Console/Program.fs",
-        "commands/dotnet-new/FSharp_Console/project.json.template"
+        "commands/dotnet-new/FSharp_Console/project.json.template",
+        "commands/dotnet-new/CSharp_Console/Program.cs",
+        "commands/dotnet-new/CSharp_Console/project.json.template",
+        "commands/dotnet-new/FSharp_Console/Program.fs",
```

Comment:
Also, please keep the list sorted, so all the files from the same template are grouped together.